docs/agave.rst

```diff
@@ -630,6 +630,29 @@ AGAVE supports the following command line options:
 
   Only valid in server mode on Linux. Allows AGAVE to run as a server on a headless cluster node.  On other platforms AGAVE must be run in a windowed desktop environment, even in server mode.
 
+.. _Citation:
+
+Citation
+--------
+
+.. |year| date:: %Y
+
+Daniel Toloudis, AGAVE Contributors (|year|). AGAVE: Advanced GPU Accelerated Volume Explorer (Version |release|) [Computer software]. Allen Institute for Cell Science. https://github.com/allen-cell-animated/agave
```

Comment:
Do we manually update the `release` placeholder with each new version release?

Reply:
Yes the release tag here picks up a version that is auto-incremented when I do releases.